ros_buildfarm/templates/status/js/setup.js

```diff
@@ -238,15 +238,24 @@ function filter_table() {
       result_rows = $.map(window.rows, function(row) {
         for (var i = 0; i < queries.length; i++) {
           var is_known_query = false;
+          var is_id_query = false;
           for(var q in QUERY_TRANSFORMS) {
-            if (queries[i] === QUERY_TRANSFORMS[q]) {
+            if (RegExp("^("+QUERY_TRANSFORMS[q]+")$").test(queries[i])) {
```

Comment:
I would rather avoid duplication since the file is big and it will be quite some effort to exact this portion into a separate file.

It's ok to keep the regexp. I was more think about impact where it isn't needed. Have you compared how it impacts e.g. the filtering on the huge Indigo page?